server/controllers/stores-controller.js

```diff
@@ -1,61 +1,64 @@
-const stores = require('./../data/stores.json')
+const stores = require("./../data/stores.json");
 
-function geoDist(lat1, lon1, lat2, lon2){
+function geoDist(lat1, lon1, lat2, lon2) {
   // script taken from: https://www.movable-type.co.uk/scripts/latlong.html
   const R = 6371e3; // metres
-  let φ1 = lat1 * Math.PI/180; // φ, λ in radians
-  let φ2 = lat2 * Math.PI/180;
-  let Δφ = (lat2-lat1) * Math.PI/180;
-  let Δλ = (lon2-lon1) * Math.PI/180;
+  let φ1 = (lat1 * Math.PI) / 180; // φ, λ in radians
+  let φ2 = (lat2 * Math.PI) / 180;
+  let Δφ = ((lat2 - lat1) * Math.PI) / 180;
+  let Δλ = ((lon2 - lon1) * Math.PI) / 180;
 
-  let a = Math.sin(Δφ/2) * Math.sin(Δφ/2) +
-            Math.cos(φ1) * Math.cos(φ2) *
-            Math.sin(Δλ/2) * Math.sin(Δλ/2);
-  let c = 2 * Math.atan2(Math.sqrt(a), Math.sqrt(1-a));
+  let a =
+    Math.sin(Δφ / 2) * Math.sin(Δφ / 2) +
+    Math.cos(φ1) * Math.cos(φ2) * Math.sin(Δλ / 2) * Math.sin(Δλ / 2);
+  let c = 2 * Math.atan2(Math.sqrt(a), Math.sqrt(1 - a));
 
   let d = R * c; // in metres
   return d;
 }
 
-exports.storesGet = async(req, res) => {
+exports.storesGet = async (req, res) => {
   let reqProps = req.body;
   let retStores = [];
   let limDist = 2000;
 
-  if (reqProps.hasOwnProperty('distance')){
+  if (reqProps.hasOwnProperty("distance")) {
     // Limit search radius
-    limDist = reqProps['distance'];
+    limDist = reqProps["distance"];
   }
   // Use searching instead
   // if (reqProps.hasOwnProperty('search')){
 
   // }
-  if (reqProps.hasOwnProperty('latitude') && reqProps.hasOwnProperty('longitude')){
-    let userLat = reqProps['latitude'];
-    let userLong = reqProps['longitude'];
-    for (let i=0;i<stores.length;++i){
-      let lat = stores[i]['latitude'];
-      let long = stores[i]['longitude'];
+  if (
+    reqProps.hasOwnProperty("latitude") &&
```

Comment:
probably need to update hasOwnProperty syntax (can new it in a new PR):
https://github.com/googleinterns/scan-and-go/issues/6